marketplace/core/types/channels/instagram/views.py

```diff
@@ -45,7 +45,11 @@ def configure(self, request, **kwargs):
                 user.email, app.project_uuid, payload, app.flows_type_code
             )
 
+            flows_config = response.get("config")
             app.flow_object_uuid = response.get("uuid")
+            app.config["title"] = response.get("name")
```

Comment:
Currently instagram saves this information in config after creating the object in flows:

            attrs["title"] = channel.get("name")
            attrs["auth_token"] = flows_config.get("auth_token")
            attrs["page_name"] = flows_config.get("page_name")
            attrs["page_id"] = flows_config.get("page_id")
            attrs["address"] = channel.get("address")

Reply:
Done